applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp

```diff
@@ -962,6 +971,10 @@ void ParMmgUtilities<PMMGLibrary::PMMG3D>::PMMGLibCallMetric(Parameters Configur
     if ( PMMG_Set_iparameter(mParMmgMesh,PMMG_IPARAM_APImode, static_cast<int>(PMMG_APIDISTRIB_nodes)) != 1 )
         KRATOS_ERROR << "Unable to set initialize parallel library through interface faces or nodes" << std::endl;
 
+    // Node global numbering: Forced to "1" all the time
+    if ( PMMG_Set_iparameter(mParMmgMesh,PMMG_IPARAM_nodeGloNum,1) != 1 )
```

Comment:
What does the "1" mean? Is it the starting global Id?

Reply:
Ok , just saw the comments on the header file for the function InitNodeGloNumParameter() 😅 

If I understand correctly it is a way to toggle the global vs local id numbering, right?

Reply:
Yes, it is just a switch to toggle it! (1==on, 0==off).
Numbering instead always starts from 1.